scheme/common/cca/platform/cca_ssd_extractor.go

```diff
@@ -1,6 +1,6 @@
-// Copyright 2022-2024 Contributors to the Veraison project.
+// Copyright 2024 Contributors to the Veraison project.
 // SPDX-License-Identifier: Apache-2.0
-package arm
+package cca
```

Comment:
The convention for naming packages (see https://go.dev/doc/effective_go#package-names) is that _"[the package] name is the base name of its source directory"_.  So this should be `platform` rather than `cca`.

Reply:
Fixed in https://github.com/veraison/services/commit/6a4041ff7758e0830e5215ba5e7ad258fa4464ea